website/addons/googledrive/utils.py

```diff
@@ -27,6 +28,7 @@ def to_hgrid(item, node, path):
         'kind': 'folder',
         'name': safe_name,
         'addon': 'googledrive',
+        'provider': 'googledrive',  # TODO: remove duplication -- APIv2
```

Comment:
Is `to_hgrid` sending things to the v2 api serializer? That seems less than ideal. You could just use the source field on the v2 serializer to tell it which field to use. Or am I misunderstanding?